docs/User-Experience/colours.tsx

```diff
@@ -0,0 +1,89 @@
+export const LEGACY = [
```

Comment:
Is it worth importing these from the `ThemeProvider/palette.ts` file so that we don't need to duplicate / update?

Reply:
The problem with `ThemeProvider/palette.ts` is that colours are stored in single variables rather than an array. To import from `ThemeProvider/palette.ts` I would need to read the file in a string format and use regex to extrapolate only the variable names and corresponding Hexadecial values.

Another issue with this approach is that I am currently working on another PR aimed to update Simorgh's colour to BBC's official colour palette, more can be found [here](https://github.com/bbc/simorgh/pull/10829/files#diff-7da1a1dc0d043db65fec729d17adb4739a322e9f42540607f15bd7e21da77a2f). From the file I just linked we would need to construct 3 different documentation which are cross platform colours, foundation colours, and service colours, making this even more challenging.
I thought about splitting the colours declaration in 3 different files, one for each palette category which would solve the above issue, there shouldn't be any negative impact in this approach other than bloating colour declaration on multiple files. 

Overall I think this is overengineering the solution. I agree that having self updating documentation would be a dream for us, but thus far whenever we update anything in our code we also make sure to update it's documentation. I would suggest to look at this as part of the documentation we should update.

I am happy to discuss further about this idea, maybe there is a quicker and easier solution and I am just missing something.